test/multi_region_aws_camunda_test.go

```diff
@@ -308,14 +360,36 @@ func deleteSecondaryRegion(t *testing.T) {
 func recreateCamundaInSecondary_8_6_plus(t *testing.T) {
 	t.Log("[C8 HELM] Recreating Camunda Platform Helm Chart in secondary 🚀")
 
-	setValues := map[string]string{
-		"operate.enabled":  "false",
-		"tasklist.enabled": "false",
+	// Check if TELEPORT is enabled.
+	teleportEnabled := false
+	if teleportStr := os.Getenv("TELEPORT"); teleportStr != "" {
+		var err error
+		teleportEnabled, err = strconv.ParseBool(teleportStr)
+		if err != nil {
+			t.Fatalf("[ELASTICSEARCH] Failed to parse TELEPORT env var: %v", err)
+		}
+	}
+
+	timeout := "600s"
```

Comment:
seeing as 2nd occurence, maybe makes sense to de-duplicate it and pull out to package level var